translations/validators.fr.yml

```diff
@@ -453,7 +453,7 @@ elected_representative.invalid_adherent: Cet e-mail correspond à un compte adh
 #
 # CertificationRequest
 #
-certification_request.document.mime_type: Seuls les fichiers jpg et png sont acceptés.
+certification_request.document.mime_type: Seuls les fichiers jpg, png et pdf sont acceptés.
```

Comment:
I think we should update this template too: 
https://github.com/EnMarche/en-marche.fr/blob/8c78625fbe725bfba31011fa8d7e2f32ff496c8c/templates/certification_request/form.html.twig#L32

Reply:
yes, I can do it in my next PR, as I will work mainly on this page